bam_mate.c

```diff
@@ -218,8 +218,39 @@ static int sync_mate(bam1_t* a, bam1_t* b)
     return 0;
 }
 
+
+static uint32_t calc_mate_score(bam1_t *b)
+{
+    uint32_t score = 0;
+    uint8_t  *qual = bam_get_qual(b);
+    int i;
+
+    for (i = 0; i < b->core.l_qseq; i++) {
+        if (qual[i] >= 15) score += qual[i];
```

Comment:
It is the quality cut-off value that is used by Picard and Biobambam for working out the mate score for duplicate marking.  Picard gets it from the samtools jdk.  I do not know why that value was chosen but it seems to be as close to a standard as we have.  There is a small comment about it in markdup where the values are used.  Proper documentation is still being written.